cmd/api/src/daemons/datapipe/cleanup.go

```diff
@@ -78,15 +79,27 @@ func (s *OrphanFileSweeper) Clear(ctx context.Context, expectedFileNames []strin
 	// Release the lock once finished
 	defer s.lock.Unlock()
 
+	log.Infof("Running OrphanFileSweeper for path %s", s.tempDirectoryRootPath)
```

Comment:
Should this be info or debug?

Reply:
I think info is appropriate since were announcing the removal of data at a given path.

Reply:
I was debating on that as well but ended up choosing info since it would go along with the "x orphan files removed" message that is also info

Reply:
I saw the lower one and was thinking it would be sufficient. But if this is gated to just on actual removal, that makes sense to me as well!

Reply:
It's not gated to removal, it runs every time. It's more just a "start of process" + location message, and the other one is the corresponding end of process message.